### ISSUE TITLE
[BUG]: Fix data folder example 

### ISSUE THREAD

**Issue by M4NU5** [AIHawk-co/Auto_Jobs_Applier#749]
### Steps to reproduce

Example config.yaml in data_folder_example/ 
There has been minor changes like jobTypes -> job_types. Below is what the config should be.  Feel free to copy paste.

```

remote: true
hybrid: true
onsite: true

experience_level:
  internship: false
  entry: true
  associate: true
  mid-senior level: true
  director: false
  executive: false

job_types:
  full_time: true
  contract: false
  part_time: false
  temporary: true
  internship: false
  other: false
  volunteer: true

date:
  all_time: false
  month: false
  week: false
  24_hours: true

positions:
  - Software engineer

locations:
  - Germany

apply_once_at_company: true

distance: 100

company_blacklist:
  - wayfair
  - Crossover

title_blacklist:
  - word1
  - word2

location_blacklist:
  - Brazil

job_applicants_threshold:
  min_applicants: 0
  max_applicants: 30

llm_model_type: openai
llm_model: 'gpt-4o-mini'
# llm_api_url: https://api.pawan.krd/cosmosrp/v1'


```

### Expected behavior

Structure should match

### Actual behavior

Structure doesnt match

### Branch

main

### LLM Used

N/A

### Model used

N/A

**Comment by feder-cr** [comment]
@M4NU5 can you work on this?

**Comment by cjbbb** [comment]
Thx, this issue should be fixed with #751

**Comment by M4NU5** [comment]
Tried to raise a pull request but couldn't
